raster_feeder/steps/config.py

```diff
@@ -24,13 +24,17 @@
 DEPTH = 78
 
 # proj inferred from sample netcdf
-NATIVE_GEO_TRANSFORM = -257.0, 2.0, 0.0, 255.0, 0.0, -2.0
+NATIVE_GEO_TRANSFORM = -256.0, 2.0, 0.0, 256.0, 0.0, -2.0
```

Comment:
Shall we ditch the whole NATIVE / WARPED prefixes and just refer to GEO_TRANSFORM and PROJECTION everywhere in init.py and rotate.py?